superset-frontend/src/explore/components/SelectAllButton/index.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { t, css, SupersetTheme } from '@superset-ui/core';
+import Button, { OnClickHandler } from 'src/components/Button';
+
+export type SelectAllButtonProps = {
+  onSelectAll: OnClickHandler;
+  selectMode: () => boolean;
+  deselectEnabled: boolean;
+};
+
+export const SelectAllButton = ({
+  onSelectAll,
+  selectMode = () => true,
+  deselectEnabled = false,
+  }: SelectAllButtonProps) =>
+  (
+    <Button
+      buttonStyle={'link'}
+      disabled={deselectEnabled ? false : !selectMode()}
+      css={(theme: SupersetTheme) =>
+        css`
+          padding: 0;
+        `
+      }
```

Comment:
```suggestion
      css={{ padding: 0 }}
```

Since `theme` is not used, this should be good enough.